samples/AzureFunctionsApp/AzureFunctionsApp.csproj

```diff
@@ -6,10 +6,10 @@
     <OutputType>Exe</OutputType>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-  
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.18.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.DurableTask" Version="1.0.2" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.19.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.DurableTask" Version="1.1.0-entities-preview.2" />
```

Comment:
`1.1.0-entities-preview.2` does not exist yet. But preview.1 is broken, so we will be publishing preview.2 eventually.